### DISCUSSION TITLE
Global search is broken?

### DISCUSSION THREAD

**Discussion by AbanoubNassem** [filamentphp/filament#4828] [Help]
There is two issues , I see ,

let's say I have two locales `['en' , 'it']`,

and I have a table `restaurant` with a record `->title` => `{"en":"Il Gancino"}`,

and my current `locale` is `it`.

when the input search is : `Gancino`.

the search query is : 

```
select * from `restaurants` where (lower(json_extract(title, "$.it")) like '%Gancino%') and `restaurants`.`deleted_at` is null limit 50
```

1st issue :- we are lowering the `column` but not the input , the query should be:

```
select * from `restaurants` where (lower(json_extract(title, "$.it")) like lower('%Gancino%')) and `restaurants`.`deleted_at` is null limit 50
```

2nd issue is , we are specifying the `column->attribute`-`locale` , this not always correct, because, what if I added the `value` of locale `en` , but not `it` , I should be able to search the `column` regardless of my current locale .

so the query should be:- 
```
select * from `restaurants` where (lower(json_extract(title, "$")) like lower('%Gancino%')) and `restaurants`.`deleted_at` is null limit 50
```

which will work , with any locale , and any `upper-lower` characters in the `search` input.

**Comment by danharrin** [comment]
Can you submit a PR to fix then?

**Reply by AbanoubNassem** [reply]
yes definitely , will do.

**Reply by AbanoubNassem** [reply]
@danharrin PR added : https://github.com/filamentphp/filament/pull/4831
